ruby/lib/ci/queue/configuration.rb

```diff
@@ -32,7 +33,7 @@ def initialize(
         timeout: 30, build_id: nil, worker_id: nil, max_requeues: 0, requeue_tolerance: 0,
         namespace: nil, seed: nil, flaky_tests: [], statsd_endpoint: nil, max_consecutive_failures: nil,
         grind_count: nil, max_duration: nil, failure_file: nil, max_test_duration: nil,
-        max_test_duration_percentile: 0.5, track_test_duration: false, max_test_failed: nil
+        max_test_duration_percentile: 0.5, track_test_duration: false, max_test_failed: nil, run_flakey_tests: 'true'
```

Comment:
`'true'` is quite ugly. The string from the ENV var should be caster to a true boolean.